### DISCUSSION TITLE
A style and scope guide for Project Pythia foundational content

### DISCUSSION THREAD

**Discussion by brian-rose** [ProjectPythia/projectpythia.github.io#50] [General]
At the Education Working Group meeting on 2/4/2021, we discussed coming up with some guidance on **style and scope** for the foundational content that Pythia will host (e.g. as discussed in #41). The idea is that individual people may find it easier to get started on building some content if they have this guidance based on group consensus:

- What do we expect from the user?
- How much time is required per unit / notebook?
- Is the content interactive, or just demonstration?

An opposing view was also expressed at the same meeting: that style can emerge from content.

I'll summarize below a few ideas that we discussed at the meeting. Between now and our next EWG meeting on 2/18/2021, we can continue to discuss here.

**Comment by brian-rose** [comment]
There was some consensus around 
- relatively short, easily digestible units
- units well organized into large topics
- Each unit has some clear labeling:
    - What this lesson will cover, 
    - List of skills you need in order to understand this lesson
    - Links to lessons on those skills as needed
- Authoring content with those cross-refs in mind will help drive development of our foundational content.

**Reply by clyne** [reply]
I don't know if I was the one that expressed the opposing view, but at least for me it would be easier to look at some content and to think about what works and what does not in terms of scope/style. Perhaps picking a topic - maybe Jupyter because I think it will be our primary content format - and putting together a draft tutorial or tutorial outline would be helpful.

**Comment by brian-rose** [comment]
We will circle back to this question as we start to populate the (tentatively named) [Pythia Foundations](https://projectpythia.org/pythia-foundations) JupyterBook collection with actual content.
